### DISCUSSION TITLE
Agenda for the Flatcar Developer Sync on Tuesday, 27th of February 2024, 3:30pm UTC

### DISCUSSION THREAD

**Discussion by t-lo** [flatcar/Flatcar#1331] [Flatcar Developer Sync]
In our monthly Flatcar developer syncs maintainers review development / roadmap progress, discuss special projects like ClusterAPI and sysext, provide an overview of ongoing work, and give an outlook of future work.

Join if you're curious about current or future Flatcar development and/or to raise visibility of a feature or bug.

- Jitsi Link: https://meet.flatcar.org/OfficeHours
- Youtube live stream / recording:  https://www.youtube.com/watch?v=Afeg22lN1SQ
  - All recordings: https://www.youtube.com/@FlatcarLinux/live
- Community Calls calendar (all future calls): [link](https://calendar.google.com/calendar/u/0/embed?src=c_ii991mqrpta9en8o7ofd4v19g4@group.calendar.google.com)
  - iCal version: [link](https://calendar.google.com/calendar/ical/c_ii991mqrpta9en8o7ofd4v19g4%40group.calendar.google.com/public/basic.ics)

## Welcome
- Brief intro/check-in of the new participants in the Jitsi call. Please introduce yourself and share what brings you here today.

## News
- Systemd-sysext PR merged: https://github.com/systemd/systemd/pull/31000
- CAPZ sysext demo'ed: https://www.youtube.com/watch?v=Fpn-E9832UQ

## Monthly Contributions Review
1. Discuss the latest [Monthly Contributions Report](https://github.com/flatcar/Flatcar/issues?q=is%3Aissue+is%3Aopen+Monthly+contributions+report+label%3Akind%2Fmetric)

## Ongoing and Planned Tasks Review
1. Roadmap review: https://github.com/orgs/flatcar/projects/7/views/9 (@t-lo)
2. Projects status review:
   a. Flatcar ClusterAPI status review (@johananl ): https://github.com/orgs/flatcar/projects/7/views/14
   b. Systemd Sysext integration status review (@pothos): https://github.com/orgs/flatcar/projects/7/views/16
      1. Mutability spec update PR: https://github.com/uapi-group/specifications/pull/78
      2. systemd PR: https://github.com/systemd/systemd/pull/31000
3. Advisories status review @dongsupark 
   a. Review of last month's advisory stats in the latest [Contributions Report](https://github.com/flatcar/Flatcar/issues?q=is%3Aissue+is%3Aopen+Monthly+contributions+report+label%3Akind%2Fmetric)
   b. Advisory status and planning: https://github.com/orgs/flatcar/projects/7/views/17
5. Maintainer team tactical / implementation (aka "line work") status review (@sayanchowdhury): https://github.com/orgs/flatcar/projects/7/views/1
6. Open Mic: Work to assign / User issues and PRs to discuss?

### Release Planning
- Ongoing and upcoming releases tracker: (https://github.com/orgs/flatcar/projects/7/views/8) (@sayanchowdhury)
- Talk about the upcoming release and the status of the packages
- Status of the ongoing release
- Preparation of the next release 

### Q&A
- Questions from community participants, answered by the Flatcar maintainers 

### Administrative tasks after meeting concluded
- [x] Create a new Github discussion for the next Flatcar developer sync: https://github.com/flatcar/Flatcar/discussions/new?category=flatcar-developer-sync
- [x] Close this discussion as "Resolved".

**Comment by vielmetti** [comment]
I'm hoping to join at the top of the hour (1600 UTC) if the meeting lasts that long.

(Edit: I will be there at 1530 UTC, a meeting got moved.)
